modules/ROOT/pages/clustering/glossary.adoc

```diff
@@ -0,0 +1,123 @@
+The cluster guarantees that a client application can read at least its own writes.
+See also <<glossary-primary, Primary server>>, <<glossary-secondary, Secondary server>>, and <<glossary-causal-consistency, Causal consistency>>.
+
+[[glossary-database]]
+| Database
+| The data store for the nodes, relationships, and properties that make up the graph.
+Multiple databases can be hosted on a Database Management Server (DBMS).
+
+[[glossary-dbms]]
+| Database Management System (DBMS)
+| The Neo4j services and system database running on an instance of a single server or cluster to provide one or more databases.
+
+[[glossary-disaster-recovery]]
+| Disaster recovery
+| A manual intervention to restore availability of a cluster, or databases within a cluster.
+
+[[glossary-election]]
+| Election
+| In the event that a leader becomes unresponsive, followers automatically trigger an election and vote for a new leader.
+A majority is required for the vote to be successful.
+
+[[glossary-fault-tolerance]]
+| Fault tolerance
+| A guarantee that a database can maintain persistence and availability in the event of one or more failures.
+The number of failures `f` that can be tolerated is dependent on the number of primaries `n` for the database and follows the formula `f = (n-1)/2`.
+In the event that more than `f` primaries fail, the database can no longer process write transactions and becomes read-only.
+
+[[glossary-follower]]
+| Follower
+| A primary copy of a database acting as a follower, receives and acknowledges synchronous writes from the leader.
+
+[[glossary-leader]]
+| Leader
+| A single Primary server is designated as the leader.
+It receives all write transactions from clients and replicates writes synchronously to followers and asynchronously to secondary copies of the database.
+Each database can have a different leader within the cluster.
+
+[[glossary-primary]]
+| Primary server
+| A primary server can be either a single instance or a core instance, both of which allow read and write operations.
+A single instance as primary is beneficial for read scalability but is not fault tolerant.
+A Core instance safeguards data and a cluster of at least three Core instances is fault tolerant.
+It participates in fault tolerant writes as it is part of the majority required to acknowledge and commit write transactions.
+
+[[glossary-read-scaling]]
+| Read scaling
+| Adding Secondary servers to the cluster can offload read queries from the Primary servers and thus reduce the load and aid write performance of the cluster.
+
+[[glossary-secondary]]
+| Secondary server
+| An asynchronously replicated instance that provides read scaling within the cluster.
+Secondary servers are made up of Read Replica instances.
+
+
+[[glossary-seed]]
+| Seed
+| A file used to create a database on a single instance or on a member of a cluster.
```

Comment:
I see this is also how it is expressed in the 5 glossary, but it feels weird to me to say the seed is *a* file. It is a bunch of files, it is is a whole store, even in different directories. I think we should leave this alone now in this PR, but return to it later, and fix it for both gloassaries.

Reply:
Agreed, this can be rephrased, but I'll leave it for now.